### PR TITLE
Ks 2021 10 add language easy german

### PR DESCRIPTION
**Pull request by Rineee**
Maybe wait for @fuzzylogic2000 s PR to be merged and then also add easy german to link texts here?

### CODE REVIEW

**Comment by Rineee** [comment]
I'll make it WIP as wating for decsiosion on 'leichte' or 'einfache' Sprache.

File: digitalstrategie/settings/base.py [hunk 1]
```diff
@@ -111,9 +112,22 @@
...
+    ('de-ls', _('Easy German'))
 ]
 
+# adding language info for easy german
+EXTRA_LANG_INFO = {
+    'de-ls': {
+        'bidi': False,
+        'code': 'de-ls',
+        'name': 'Easy German',
+        'name_local': u"Leichte Sprache",
```
Comment (goapunk) [inline digitalstrategie/settings/base.py:125]: do we really need the `u` ?

**Review by fuzzylogic2000 — COMMENTED** [review]
Nice!

File: digitalstrategie/settings/base.py (hunk 1 above)
Comment (Rineee) [reply digitalstrategie/settings/base.py:125]: I found somewhere that it needs to be unicode and in Aplus we also did it like that.. but to be honest not sure if that is needed, it also works without.. should I remove it you think?

**Comment by Rineee** [comment]
Ok, I think we could merge this now as the name_locale could be changed easily afterwards ..?

File: digitalstrategie/settings/base.py (hunk 1 above)
Comment (goapunk) [reply digitalstrategie/settings/base.py:125]: I don't really know about it in a Django context, but afaik it was required in python2 - in python3 this become unnecessary, so yes, if it works I think we should remove it

Comment (Rineee) [reply digitalstrategie/settings/base.py]: Done!

**Review by fuzzylogic2000 — APPROVED** [review]
Yeah!

**Pull request merged**